formal/data_ind_timing/Makefile

```diff
@@ -0,0 +1,38 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+# A simple wrapper around fusesoc to make it a bit easier to run the formal flow
+
+core-name   := lowrisc:fpv:ibex_data_ind_timing
+vlnv        := $(subst :,_,$(core-name))
+build-root  := $(abspath ../../build/$(vlnv))
+
+# Since we have a hacky hook that runs sv2v in place on fusesoc's
+# copied source files, we have to generate different build roots for
+# the two flavours (otherwise bad things will happen if you run make
+# -j2)
+mk-build-root = $(abspath ../../build/$(vlnv)/$(1))
+
+mk-fusesoc-cmd = \
+  fusesoc --cores-root=../.. \
+  run --build-root=$(call mk-build-root,$(1)) \
+  --target=$(1) \
+  $(core-name) $(fusesoc-params)
+
+.PHONY: all slow fast single
+all: slow fast single
+
+slow: slow_mull slow_mulh slow_div slow_rem
+fast: fast_mull fast_mulh fast_div fast_rem
+single: single_mull single_mulh single_div single_rem
+
+slow_mull slow_mulh slow_div slow_rem:
+	$(call mk-fusesoc-cmd,$@)
+fast_mull fast_mulh fast_div fast_rem:
+	$(call mk-fusesoc-cmd,$@)
+single_mull single_mulh single_div single_rem:
+	$(call mk-fusesoc-cmd,$@)
+
+lint:
+	mypy --strict sv2v_in_place.py
```

Comment:
I don't think you want this bit :-) Also, could you move that target from the icache directory in the previous commit?

Reply:
Yeah, that does not make much sense :) Thanks for spotting that.

Moving to where? The script is not part of a make system anymore. Would it be okay to just remove this target altogether as it seems to be a specific developer only target?

Reply:
Could you just stick a Makefile with this lint target in the same directory as the script? I agree that it's somewhat developer-only, but I dream of a day when we enforce these lint checks in CI :-)

Reply:
Okay, will find a nice place for this :)
True, CI should do something like this. We can sneak in a github action super-linter ;)

Reply:
Added a target in the root Makefile. With a check for all python files in the util folder. Two of three files report some errors.